src/ui-kit/Button/Button.styles.js

```diff
@@ -78,41 +78,63 @@ const buttonTypeProp = ({ theme, type }) => {
           border: ${darken(theme.colors.text.secondary, 0.06)};
         }
       `;
-    case 'link':
+    case "cta":
+      return css`
+        background: #00b2eb;
+        border: ${theme.colors.base.secondary};
+        &:focus,
+        &:hover {
+          border: ${darken(theme.colors.text.secondary, 0.06)};
+        }
+      `;
+    case "link":
       return css`
         border-width: 2px;
         background: none;
       `;
   }
 };
 
-const buttonSizeProp = ({ size }) => {
+const buttonSizeProp = ({ size, ...props }) => {
+  console.log(props);
   switch (size) {
     default:
-    case 'large':
+    case "large":
       return css`
-        padding: ${themeGet('space.xs')} ${themeGet('space.l')};
+        padding: ${themeGet("space.xs")} ${themeGet("space.l")};
       `;
-    case 'small':
+    case "small":
       return css`
-        padding: ${themeGet('space.xxs')} ${themeGet('space.base')};
+        padding: ${themeGet("space.xxs")} ${themeGet("space.base")};
       `;
-    case 'micro':
+    case "micro":
       return css`
-        padding: ${themeGet('space.xxs')} ${themeGet('space.xs')};
+        padding: ${themeGet("space.xxs")} ${themeGet("space.xs")};
+      `;
+    case "button-feature":
```

Comment:
Instead of doing a specific case for "button-feature" we should just have our buttons fill their container and then add a max-width on a parent container if we want to limit their size on larger screens.

Reply:
That makes sense. I removed the case, and just have the Button have 100% width